onchain/src/art_peace.cairo

```diff
@@ -1262,6 +1263,22 @@ pub mod ArtPeace {
         }
     }
 
+    #[abi(embed_v0)]
+    impl ArtPeaceCanvasNFTLikeAndUnlike of ICanvasNFTLikeAndUnlike<ContractState> {
+        fn like_nft(ref self: ContractState, token_id: u256) {
+            let nft_address = self.nft_contract.read();
+            ICanvasNFTLikeAndUnlikeDispatcher { contract_address: nft_address }.like_nft(token_id);
+            let nft_owner = IERC721Dispatcher { contract_address: nft_address }.owner_of(token_id);
+
+            // award the minter of the nft 1 extra pixel each time someone likes the nft
+            self.extra_pixels.write(nft_owner, self.extra_pixels.read(nft_owner) + 1);
+        }
```

Comment:
Also, we will want to emit an event indicating the user obtained extra pixels.
We should probably create a dedicated event like "ExtraPixelsAwarded" or something with the users address & amount of pixels. Then we can emit that event here.

We probably want to change "HostAwardedUser", "MainQuestClaimed", and "DailyQuestClaimed" events to this new event too so it is consistent.